### DISCUSSION TITLE
Missing links in mobile Accounts tab

### DISCUSSION THREAD

**Discussion by sciencell-lab** [breezefront/community#190] [1. Help]
Hi
When I enable Breeze, the links pop up in mobile when I disable header builder in my theme editor for argento-pure2.
When I disable Breeze, the links pop up even with header builder enabled. 

Do you know what/or where this issue is coming from by any chance? Is this an error coming from my theme or my theme editor?

**Comment by dimdimich** [comment]
Please open ticket at https://argentotheme.com/contacts in case you are using Argento theme.
